core-features/generate-client-and-cli-install-no-lockfile-npm/test.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+set -eux
+
+if [ ! -f "package-lock.json" ]; then
+  echo "Test should have produced a package-lock.json file"
+  exit 1
+fi
```

Comment:
before running test?

Reply:
Yes, the test check the contents of the package.json and this looks that only the correct lockfile was created by generate.